src/libstore-tests/data/derivation/ca/advanced-attributes-defaults.json

```diff
@@ -15,6 +15,27 @@
   "inputDrvs": {},
   "inputSrcs": [],
   "name": "advanced-attributes-defaults",
+  "options": {
+    "additionalSandboxProfile": "",
+    "allowLocalNetworking": false,
+    "allowSubstitutes": true,
+    "impureEnvVars": [],
+    "impureHostDeps": [],
+    "noChroot": false,
+    "outputChecks": {
+      "forAllOutputs": {
+        "allowedReferences": null,
+        "allowedRequisites": null,
+        "disallowedReferences": [],
+        "disallowedRequisites": [],
+        "ignoreSelfRefs": true
+      }
+    },
+    "passAsFile": [],
+    "preferLocalBuild": false,
+    "requiredSystemFeatures": [],
+    "unsafeDiscardReferences": {}
```

Comment:
So I would, contra our usual JSON policies, *not* want unused options in here. This is because I want to think of options as an "*extensible* record", where various Nix implementations are e.g. free to do their own custom options under their own prefix, and whenever an unknown option is encountered, as sensible warning is given.